docs/docs/comment_dsl.mdx

```diff
@@ -104,6 +104,61 @@ foo = uint ; @newtype @custom_json
 
 Avoids generating and/or deriving json-related traits under the assumption that the user will supply their own implementation to be used in the generated library.
 
+## @custom_serialize / @custom_deserialize
```

Comment:
I think it's not necessarily obvious from the docs that what comes after these is a user-specified arbitrary string (i.e. `write_hex_string` is not some magical thing built into cddl-codegen, but rather an arbitrary string we just used as an example)

Reply:
I added a short paragraph explaining that the string is just directly called and thus must be qualified or imported.